feat(experiments): add Analyzing results docs page (#12788)
PostHog/posthog.com: contents/docs/experiments/analyzing-results.mdx

sidebar: Docs
showTitle: true
availability:
  free: none
  selfServe: full
  enterprise: full
---

import {ProductScreenshot} from 'components/ProductScreenshot'

export const ExposuresPanelLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/exposures_light_04d3ee6606.png"
export const ExposuresPanelDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/exposures_dark_6cf47da9eb.png"
export const DeltaChartLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/delta_chart_light_3010641eff.png"
export const DeltaChartDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/delta_chart_dark_82a4470a98.png"
export const DeltaChartHoverLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/popup_light_8fe379428f.png"
export const DeltaChartHoverDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/popup_dark_cd122710e7.png"
export const FunnelBreakdownLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/funnel_light_e933e07d24.png"
export const FunnelBreakdownDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/funnel_dark_c6a7c3e830.png"

> Once your experiment has been running and gathering data, you'll want to analyze the results to determine if your changes had a statistically significant impact. This guide walks you through understanding and interpreting experiment results in PostHog.

Reviewer: ```suggestion
After your experiment runs and collects data, you'll want to analyze the results to determine if your changes had a statistically significant impact. This guide shows you how to understand and interpret your experiment results in PostHog.
```

Clarity

## Understanding the results page

### Exposures panel

The exposures panel shows you how many users have been exposed to each variant of your experiment:

<ProductScreenshot
    imageLight={ExposuresPanelLight} 
    imageDark={ExposuresPanelDark}
    alt="Experiment exposures panel showing user distribution across variants" 
    classes="rounded"
/>

This collapsible panel displays:
- **Exposure count**: The total number of unique users who have seen each variant
- **Percentage distribution**: How users are split between variants (typically 50/50 for two variants)

The exposure numbers help you verify that:
1. Your experiment is running and users are being assigned to variants
2. The split between variants matches your expected allocation
3. You have sufficient sample size for meaningful results

### Metric results and significance

For each metric you're tracking, PostHog displays a delta chart showing the performance of each variant compared to the control:

<ProductScreenshot
    imageLight={DeltaChartLight} 
    imageDark={DeltaChartDark}
    alt="Delta chart showing experiment results with confidence intervals" 
    classes="rounded"
/>

> ## How to determine if a variant is significant

Reviewer: Should we redo the heading heirarchy in this page to be:
H2: Understanding the results page
   -> This will only cover the exposures panel
 
Also H2: Metric results and significance
    -> We combine this with the How to determine if a variant is significant
    
I think this might be slightly better, the important section on significance is disjoint from the diagram/explanation above. I think I'm okay with breaking up the explanation of the results page for a better focus on significance

Author: I think I addressed this, please check and feel free to adjust :)

The most important question when analyzing results is: **"Is my variant statistically significant?"**

Look for these clear visual indicators in the delta chart:

1. **Color-coded deltas**: When a variant achieves statistical significance:

>    - **Green with ↑ arrow**: The variant is winning (positive significant change)

Reviewer: Does this language still make sense given that experiments support goals for numbers to go ⬆️  and ⬇️  now?

   - **Red with ↓ arrow**: The variant is losing (negative significant change)
   - **No color**: The result is not statistically significant

2. **Confidence/Credible intervals**: The horizontal bars show the 95% confidence interval (frequentist) or credible interval (Bayesian):
   - If the interval doesn't cross zero, the result is statistically significant
   - The narrower the interval, the more precise your estimate

3. **Hover details**: When you hover over a variant in the delta chart, you'll see detailed statistics:

> 

Reviewer: Do we have resources to point people to who want a quick refresher on basic stats? :P

<ProductScreenshot
    imageLight={DeltaChartHoverLight} 
    imageDark={DeltaChartHoverDark}
    classes="max-w-40"
    alt="Detailed statistics shown when hovering over a variant" 
    classes="rounded"
/>

The hover popup shows:
- **Significance status**: States if the variant "won", "lost", or is "not significant"
- **Total value**: The actual measured value for this metric (e.g., mean for trends, conversion rate for funnels)
- **Exposures**: Number of users exposed to this variant (matches the exposures panel)

> - **Chance to win** (Bayesian) or **p-value** (Frequentist): Statistical confidence measure

Reviewer: I think we can hyper link out to https://posthog.com/docs/experiments/frequentist-method and other pages. I remember we have some pages on these. good to crossl ink.

## Interpreting different scenarios

### Clear winner

When all your primary metrics show green (positive) significance for a variant:
- You have strong evidence that the variant improves your key metrics
- Consider the magnitude of improvement – even statistically significant changes might not be practically significant
- Review secondary metrics to ensure no unexpected negative impacts

### Mixed results

It's common to see some metrics improve while others decline. When this happens:

1. **Prioritize your primary metric**: Focus on the metric you defined as most important when setting up the experiment
2. **Understand the trade-offs**: 
   - Is a 5% increase in conversion worth a 2% decrease in average order value?
   - Does improved engagement justify slightly lower retention?
3. **Dig deeper with analytics**:
   - Break down results by user segments to understand who benefits and who doesn't

>    - Use session replays to see how users interact differently with each variant

Reviewer: Link out to product docs for stuff we mention here :D

Author: Added!

   - Create cohorts of users from each variant for further analysis
4. **Consider the user journey**: Sometimes negative changes in one metric lead to positive downstream effects

### No significant results

If your experiment shows no significant differences:
- **Check sample size**: You might need more data. Use the sample size calculator to estimate required duration
- **Verify implementation**: Ensure the variants are actually different and the tracking is working correctly
- **Consider effect size**: The actual difference might be too small to detect with your current sample size

> - **Learn from it**: Not all experiments win – this is valuable information about what doesn't move the needle

Reviewer: ```suggestion
- **Learn from it**: Not all experiments win – this is valuable information about what doesn't move the needle
```
